examples/quantization_aware_training/torch/resnet18/main.py

```diff
@@ -262,8 +266,32 @@ def transform_fn(data_item):
     criterion = nn.CrossEntropyLoss().to(device)
     compression_lr = 1e-5
     optimizer = torch.optim.Adam(quantized_model.parameters(), lr=compression_lr)
-
-    train(train_loader, quantized_model, criterion, optimizer, device=device)
+    acc1_int8_best = 0.0
+
+    for epoch in range(TRAINING_EPOCHS):
+        print(f"Train epoch: {epoch}")
+        train_epoch(train_loader, quantized_model, criterion, optimizer, device=device)
+        acc1_int8 = validate(val_loader, quantized_model, device)
+        print(f"Accyracy@1 of INT8 model after {epoch} epoch finetuning: {acc1_int8:.3f}")
+        # Save the compression checkpoint for model with the best accuracy metric.
+        if acc1_int8 > acc1_int8_best:
+            state_dict = quantized_model.state_dict()
+            compression_config = quantized_model.nncf.get_config()
+            torch.save(
+                {
+                    "model_state_dict": state_dict,
+                    "compression_config": compression_config,
+                },
+                ROOT / BEST_CKPT_NAME,
+            )
+            acc1_int8_best = acc1_int8
+
+    # Load best checkpoint
```

Comment:
```suggestion
    # Load best checkpoint to source model with applying quantization transformations 
```
Need to describe that it's not just load state in torch.